src/remediations/controller.read.js

```diff
@@ -27,6 +27,33 @@ const catchErrorCode = (code, fn) => e => {
     throw e;
 };
 
+async function fetchRemediation (req) {
+    const id = req.params.id;
+    const hosts = req.query.hosts;
```

Comment:
what happens if no `hosts` query param is provided?

Reply:
@jharting At the moment should no "hosts" param be provided and the /playbook method be called with cert-auth the operation returns a 404 error due too the owner_id check not being able to run.  Would it make more sense for this to be a 400?

Reply:
I am wondering if in that case we should return the playbook but only as long as the only host used in the playbook is the one that matches the cert. Or perhaps automatically return a slice that matches the given cert?